### DISCUSSION TITLE
COP reset to 0

### DISCUSSION THREAD

**Discussion by razorclam** [gekkekoe/esphome-ecodan-hp#81] [General]
Hi,
I noticed the DHW COP reset to 0 at midnight. Assuming it resets each month?
However the heating COP did not reset. Is this expected behaviour?

**Comment by gekkekoe** [comment]
it's the cop of the previous day, so there was probably no dhw run yesterday...

**Reply by razorclam** [reply]
Perfect. That ties up. Thanks
